src/screens/CItyScreen/CityScreen.tsx

```diff
@@ -41,11 +45,14 @@ export const CityScreen = () => {
           ) : (
             <View style={styles.cardContainer}>
               <Text style={styles.titleText}>{defaultCity.city}</Text>
+              <Text>{currentDay}</Text>
               <WeatherCardDayTemplate
-                day={current_Day}
+                description={dayWeatherInfo.weather[0].description}
```

Comment:
Please, destruct above.

Reply:
conditional rendering doesnt allow me to make it correct way (only using local state, but I think this is worse)